boards/sensry/ganymed_bob/doc/index.rst

```diff
@@ -28,117 +28,101 @@ The SoC has the following core features:
 * 512 KB Global MRAM
 * 512 KB Secure MRAM
 * CLOCK
-* RAM
-* :abbr:`32x GPIO (General Purpose Input Output)`
-* :abbr:`4x TWIM (I2C-compatible two-wire interface with MicroDMA)`
-* 4x I2S
-* :abbr:`7x SPI (Serial Peripheral Interface with MicroDMA)`
-* :abbr:`3x UART (Universal receiver-transmitter with MicroDMA)`
-* :abbr:`1x TSN (Time sensitive networking ethernet MAC with MicroDMA)`
-* 1x CAN-FD
-* 3x ADC
-
-.. figure:: img/ganymed_bob_sy120_gbm.webp
-     :align: center
-     :alt: Ganymed Break-Out-Board (BOB) equipped with SY120 GBM
-
-     Ganymed Break-Out-Board (BOB) equipped with SY120 GBM (Credit: Sensry)
+* Peripherals
+
+    * :abbr:`32x GPIO (General Purpose Input Output)`
+    * :abbr:`4x TWIM (I2C-compatible two-wire interface with MicroDMA)`
+    * 4x I2S
+    * :abbr:`7x SPI (Serial Peripheral Interface with MicroDMA)`
+    * :abbr:`3x UART (Universal receiver-transmitter with MicroDMA)`
+    * :abbr:`1x TSN (Time sensitive networking ethernet MAC with MicroDMA)`
+    * 1x CAN-FD
+    * 3x ADC
+* Power section for on-board power generation and power measurement (selectable)
+
+    * USB type-C
+    * external 5V power source
+* 40-pin JTAG connector (compatible to Olimex ARM-JTAG-OCD-H)
+* USB over FTDI (connected to UART0)
+* Header for all I/Os and configuration
 
-.. figure:: img/ganymed_bob_sy120_gen1.webp
-     :align: center
-     :alt: Ganymed Break-Out-Board (BOB) equipped with SY120 GEN1
+* Assembly options for the SoC
 
-     Ganymed Break-Out-Board (BOB) equipped with SY120 GEN1 (Credit: Sensry)
+  * SY120-GBM - Generic Base Module without top level sensors
+  * SY120-GEN1 - Generic Module type 1 with top level sensors (see below)
+
+The ``ganymed-bob/sy120-gen1`` comes with additional hardware:
+
++-----------+------------+----------------------------+
+| Interface | Controller | Driver/Component           |
++===========+============+============================+
+| BME680    | on-chip    | environment sensor (SPI1)  |
++-----------+------------+----------------------------+
+| BMA456    | on-chip    | acceleration sensor (SPI0) |
++-----------+------------+----------------------------+
+| BMG250    | on-chip    | gyrosope sensor (SPI2)     |
++-----------+------------+----------------------------+
+| MIS2DH    | on-chip    | vibration sensor (I2C0)    |
++-----------+------------+----------------------------+
```

Comment:
I would just let the .. zephyr:board-supported-hw:: directive take care of this for you.

```suggestion

The ``ganymed-bob/sy120-gen1`` comes with additional on-board sensors.
```